src/datetime/rustc_serialize.rs

```diff
@@ -0,0 +1,119 @@
+use super::DateTime;
+#[cfg(feature = "clock")]
+use crate::offset::Local;
+use crate::offset::{FixedOffset, LocalResult, TimeZone, Utc};
+use core::fmt;
+use core::ops::Deref;
+use rustc_serialize::{Decodable, Decoder, Encodable, Encoder};
```

Comment:
Why use rustc_serialize? Serde?

Reply:
I'm not making any backwards incompatible changes here, so that means keeping rustc_serialize.